src/coin_test/backtest/backtest_results.py

```diff
@@ -104,3 +104,22 @@ def value_from_portfolio(t: pd.Timestamp, p: Portfolio, c: Composer) -> float:
             total += conversion * money.qty
 
         return total
+
+    @staticmethod
+    def load(fp: str) -> "BacktestResults":
+        """Load BacktestResults from disk.
+
+        Args:
+            fp: filepath to pickle file to load from
+
+        Returns:
+            BacktestResults: BacktestResults stored at the location
+
+        Raises:
+            ValueError: raises ValueError if the specified file path is not a file
+        """
+        if not os.path.isfile(fp):
```

Comment:
maybe add an info log here about loading backtestresults, or after it successfully loads?